README.md

```diff
@@ -160,18 +134,21 @@ cargo build --features hot-reload
 copy target/debug/dcs_grpc.dll target/debug/dcs_grpc_hot_reload.dll
 ```
 
-### Prepare Mission
+### Prepare DCS
 
-For development, update the previously added line in `DCS World\Scripts\MissionScripting.lua` to point to your checked out repository of the gRPC server:
+For development:
 
-```diff
-- dofile(lfs.writedir()..[[Scripts\DCS-gRPC\grpc-mission.lua]])
-+ GRPC = {
-+ 	dllPath = [[C:\Development\DCS-gRPC\rust-server\target\debug\]],
-+ 	luaPath = [[C:\Development\DCS-gRPC\rust-server\lua\DCS-gRPC]]
-+ }
-+ dofile(GRPC.luaPath .. [[grpc-mission.lua]])
-```
+- update your `MissionScripting.lua` to load `grpc-mission.lua` from your local clone, e.g.:
+  ```diff
+  - dofile(lfs.writedir()..[[Scripts\DCS-gRPC\grpc-mission.lua]])
+  + dofile([[C:\Development\DCS-gRPC\rust-server\lua\DCS-gRPC\grpc-mission.lua]])
```

Comment:
Only drawback of this PR is that you cannot simply do `dofile(GRPC.luaPath .. [[grpc-mission.lua]])` anymore, since `GRPC.luaPath` isn't loaded yet.

Reply:
I think this is forgivable for a development workflow. :) (I don't mind because I appreciate that LUA environments are really segregated)

Alternatively, we could make the syntax ugly and do a dirty load on `dofile` load.

Another option is to advise the developer to do a symbolic link on the lua part of the app. https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/mklink